source/Octopus.Tentacle.Client/Execution/RpcCallExecutor.cs

```diff
@@ -89,28 +90,67 @@ internal RpcCallExecutor(RpcCallRetryHandler rpcCallRetryHandler, ITentacleClien
             Func<CancellationToken, T> action,
             ClientOperationMetricsBuilder clientOperationMetricsBuilder,
             CancellationToken cancellationToken)
+        {
+            return Execute(rpcCall, action, abandonActionOnCancellation: false, clientOperationMetricsBuilder, cancellationToken);
+        }
+
+        public T Execute<T>(
+            RpcCall rpcCall,
+            Func<CancellationToken, T> action,
+            bool abandonActionOnCancellation,
+            ClientOperationMetricsBuilder clientOperationMetricsBuilder,
+            CancellationToken cancellationToken)
         {
             var rpcCallMetricsBuilder = RpcCallMetricsBuilder.StartWithoutRetries(rpcCall);
             var start = DateTimeOffset.UtcNow;
+            var abandonAfter = TimeSpan.FromSeconds(5);
```

Comment:
Can we make this a field so both `Execute` and `ExecuteWithRetries` use the same value and don't drift